package.json

```diff
@@ -37,5 +38,8 @@
     "ts-jest": "^23.1.4",
     "tslint": "^5.11.0",
     "typescript": "^3.0.1"
-  }
+  },
+  "files": [
+    "default-htmltemplate.hbs"
```

Comment:
Did you observe that this wasn't included if it wasn't specified here? It shows up if I run `npm pack`, at least. I'd like to avoid maintaining the `files` array as it changes the behavior of how things get packaged and needs to be updated for certain kinds of changes.

Reply:
makes sense. probably don't need it.